obtainEstimatedWrenches.m

```diff
@@ -106,12 +120,36 @@
 % Print the unknowns to make sure that everything is properly working
 %fullBodyUnknowns.toString(estimator.model())
 
+% Prepare the datastructure that we used in the loop outside the loop 
+% (to improve performances, as otherwise we allocate memory at each 
+% loop step, and allocating memory is usually quite a slow operation) 
 qj_idyn   = iDynTree.JointPosDoubleArray(dofs);
 dqj_idyn  = iDynTree.JointDOFsDoubleArray(dofs);
 ddqj_idyn = iDynTree.JointDOFsDoubleArray(dofs);
 
+    
+% The estimated FT sensor measurements
+estFTmeasurements = iDynTree.SensorsMeasurements(estimator.sensors());
+    
+% The estimated joint torques
+estJointTorques = iDynTree.JointDOFsDoubleArray(dofs);
+    
+% The estimated contact forces
+estContactForces = iDynTree.LinkContactWrenches(estimator.model());
+
+% Sensor wrench buffer 
+estimatedSensorWrench = iDynTree.Wrench();
+
+
 %% For each time instant
-for t=1:size(resampledTime)
+fprintf('obtainEstimatedWrenches: Computing the estimated wrenches\n');
+for t=1:length(resampledTime)
```

Comment:
the change form size to length was due to speed performance improvement? In that case maybe we can try getting the length before the for apparently is even better.
It's what it says here :
http://www.mathworks.com/matlabcentral/fileexchange/48038-size---vs-length---vs-storing-length-value-before-entering-for-loop